templates/community-viz/package.json

```diff
@@ -1,17 +1,19 @@
 {
   "config": {
-    "gcsDevBucket": "{{DEV_BUCKET}}",
+    "gcsDevBucket": "gs://yulanlin-cv-dev/template",
     "gcsProdBucket": "{{PROD_BUCKET}}",
     "jsFile": "index.js",
     "jsonFile": "index.json",
     "cssFile": "index.css",
+    "print": "printMessage.js",
     "manifestFile": "manifest.json"
   },
   "scripts": {
     "build:dev": "node scripts/build.js --prod false",
     "build:prod": "node scripts/build.js --prod true",
     "push:dev": "node scripts/push.js --prod false",
     "push:prod": "node scripts/push.js --prod true",
+    "message": "node scripts/message.js --prod false",
```

Comment:
This isn't a very descriptive name. Could we call this something like 'getMessage' or 'getMessageMock` or something?